olp-cpp-sdk-dataservice-read/include/olp/dataservice/read/VersionedLayerClient.h

```diff
@@ -382,6 +382,32 @@ class DATASERVICE_READ_API VersionedLayerClient final {
    */
   bool IsCached(const geo::TileKey& tile) const;
 
+  /**
+   * @brief Protects tile keys from eviction.
+   *
+   * @note Only tiles which data handled are present in the cache at the time of
+   * the call can be protected
+   *
```

Comment:
Please add an additional `@note` specifying that this should not be called while Release() call is in progress.